site/ru/blog/new-in-devtools-96/index.md

```diff
@@ -29,170 +28,235 @@ draft: true
 {% include 'partials/devtools/ru/banner.md' %}
 
 <!-- ## Preview feature: New CSS Overview panel {: #css-overview } -->
+## Ранний доступ: Новая вкладка Обзор CSS {: #css-overview }
 
 <!-- Use the new **CSS Overview** panel to identify potential CSS improvements on your page.
 [Open the **CSS Overview** panel](/docs/devtools/css-overview#open), then click on **Capture overview** to generate a report of your page’s CSS. -->
+Используйте новую вкладку **Обзор CSS** (CSS Overview) чтобы выявить возможные улучшения CSS на вашем сайте.
+[Откройте вкладку **Обзор CSS** (CSS Overview)](/docs/devtools/css-overview#open), затем кликните на кнопку **Capture overview** чтобы сгенерировать отчёт о CSS на вашей странице.
 
 <!-- You can further drill down on the information. For example, click on a color in the **Colors** section to view the list of elements that apply the same color. Click on an element to open the element in the **Elements** panel. -->
+Вы можете подробней изучить информацию. Например, кликните на цвет в секции **Цвета** чтобы увидеть список всех элементов, к которым применяется выбранный цвет. Кликните на элемент чтобы увидеть его на вкладке **Элементы** (Elements).
+
 
 <!-- The **CSS Overview** panel is a preview feature. Our team is still actively working on it and we are looking for your [feedback](https://goo.gle/css-overview-feedback) for further enhancements. -->
+Вкладка **Обзор CSS** (CSS Overview) пока в раннем доступе. Наша команда все еще активно работает над ней, и мы ждем [ваши отзывы](https://goo.gle/css-overview-feedback) для дальнейших улучшений.
 
 <!-- Read [this article](/docs/devtools/css-overview) to learn more on the **CSS Overview** panel. -->
+Читайте [эту статью](/docs/devtools/css-overview) чтобы узнать больше о вкладке **Обзор CSS** (CSS Overview).
 
-{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/fXXPihV3bTl82WDJGX51.png", alt="CSS Overview panel", width="800", height="509" %}
+{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/fXXPihV3bTl82WDJGX51.png", alt="Вкладка Обзор CSS", width="800", height="509" %}
 
 {# https://chromium.googlesource.com/devtools/devtools-frontend/+/ef26abc89035075bbdb08f1b26c1b8fd942ffc04 #}
 
-Задачи в трекере Chromium: [1254557](https://crbug.com/1254557)
+Задача в трекере Chromium: [1254557](https://crbug.com/1254557)
+
+
+<!-- ## Restored and improved CSS length edit and copy experince {: #length } -->
```

Comment:
I added a new section here as we merged some fixes in M96 after my article is published. Please help to translate as well. Thanks!